src/shared/report_op.c

```diff
@@ -522,7 +522,7 @@ void os_ReportdStart(report_filter *r_filter)
                     mgroup++;
                 }
 
-                free(mgroup);
+                //free(mgroup);
```

Comment:
I just used the terminology from the report. That free was making it crash, removing it stopped the crash. I can provide gdb output later.